Content.Shared/DrawDepth/DrawDepth.cs

```diff
@@ -9,7 +9,16 @@ public enum DrawDepth
         /// <summary>
         ///     This is for sub-floors, the floors you see after prying off a tile.
         /// </summary>
-        LowFloors = DrawDepthTag.Default - 6,
+        LowFloors = DrawDepthTag.Default - 10,
+
+        // various entity types that require different
+        // draw depths, as to avoid hiding
+        #region SubfloorEntities
+        ThickPipe = DrawDepthTag.Default - 9,
+        ThickWire = DrawDepthTag.Default - 8,
+        ThinPipe = DrawDepthTag.Default - 7,
+        ThinWire = DrawDepthTag.Default - 6,
+        #endregion
 
         /// <summary>
         ///     Things that are beneath regular floors, such as wires or pipes. vents/scrubbers also use this to ensure
```

Comment:
Maybe update the description here, seeing as this isn't used by wires/pipes anymore.
Also vents and such use FloorObjects now, so this is already out of date (my bad).

I think BelowFloor is now only used by `CableTerminal`, but it can probably stick around just in case someone wants something above wires & pipes but below floors.